src/containers/donation/Donation.jsx

```diff
@@ -0,0 +1,268 @@
+                    sed do eiusmod tempor incididunt ut labore et dolore magna
+                    aliqua. Quis ipsum suspendisse ultrices gravida. Risus
+                    commodo viverra maecenas accumsan lacus vel facilisis.
+                  </p>
+                </div>
+
+                <div className="lightGreySection">
+                  <img
+                    src="https://i.ibb.co/LDKPgmn/gift-icon.png"
+                    alt="Gift icon"
+                  />
+                  <p className="lightGreyColumn">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                    do eiusmod tempor incididunt ut labore et dolore magna
+                    aliqua. Quis ipsum suspendisse ultrices gravida. Risus
+                    commodo viverra maecenas accumsan lacus vel facilisis.
+                  </p>
+                </div>
+
+                <div className="darkGreySection">
+                  <img
+                    src="https://i.ibb.co/85qjgNG/heart-icon.png"
+                    alt="Heart icon"
+                  />
+                  <p className="darkGreyColumn">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                    do eiusmod tempor incididunt ut labore et dolore magna
+                    aliqua. Quis ipsum suspendisse ultrices gravida. Risus
+                    commodo viverra maecenas accumsan lacus vel facilisis.
+                  </p>
+                </div>
+
+                <div className="lightGreySection">
+                  <img
+                    src="https://i.ibb.co/34RCGp3/earth-icon.png"
+                    alt="Earth icon"
+                  />
+                  <p className="lightGreyColumn">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                    do eiusmod tempor incididunt ut labore et dolore magna
+                    aliqua. Quis ipsum suspendisse ultrices gravida. Risus
+                    commodo viverra maecenas accumsan lacus vel facilisis.
+                  </p>
+                </div>
+              </div>
+            </section>
+          </Col>
+        </Row>
+
+        <Row>
+          <Col xs={12} sm={12} md={12} lg={12}>
+            <section className="helpParagraph">
+              <h3>How can you help us?</h3>
+              <article>
+                Approximately 11 people in Canada die by suicide every day.
+                That’s over 4,000 people a year, making suicide a leading cause
+                of death in Canada. We can help prevent suicide with the right
+                services. A donation to the Canada Suicide Prevention Service
+                allows us to connect individuals in distress and individuals
+                with suicidal thoughts with local Crisis Centres to receive
+                immediate support.
+                <br />
+                Approximately 11 people iight services. A donation to the Canada
```

Comment:
i think theres a typo on this line with iight

Reply:
It's just a lorem ipsum as a suggestion what shall we add into this part

Reply:
@louisrli If everything is Ok shall I merge?